testing/packer/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: 7heo <7heo@mail.com>
+# Maintainer: 7heo <7heo@mail.com>
+pkgname=packer
+pkgver=0.10.2
+pkgrel=0
+pkgdesc="Packer is a tool for creating identical machine images for multiple platforms from a single source configuration."
+url="https://www.packer.io/"
+arch="x86_64"
+license="MPL 2.0"
+depends=""
+makedepends="go go-tools"
+install=""
+subpackages=""
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/mitchellh/$pkgname/archive/v$pkgver.tar.gz"
+
+builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd $srcdir
+	export GOPATH="$startdir"
```

Comment:
I would personally prefer the approach that @gena01 used in https://github.com/alpinelinux/aports/commit/5bef15666ec397f281fa09fa57b059549902f144#diff-09aa8ce633e2619a7d2132841839fce1R19, it looks less hack-ish for me. WDYT?